cubes/new_cube.py

```diff
@@ -0,0 +1,254 @@
+""" JXP VERSION """
+
+'''
+Questions for Clancy:
+
+1. lC vs. C
+2. Order of update_grid()
+3. Survey is unique to a grid, right?
+4. Order of grid parameters
+5. Why 90s per iteration?
+6. Is it ok to get a NAN??   And why is it ignored??
+7. https://docs.scipy.org/doc/scipy/reference/tutorial/integrate.html#faster-integration-using-low-level-callback-functions
```

Comment:
In a magical future where I don't need to sleep and there are 50 hr per day I will read this.